### DISCUSSION TITLE
How do you open the Buffer Symbols search?

### DISCUSSION THREAD

**Discussion by tkfv** [zed-industries/zed#13971] [Feature Requests]
There is a Project Symbols Toggle in the command palette, but I can't find the Buffer Symbols search.

And yet it seems to exists by looking at this issue: https://github.com/zed-industries/zed/issues/4999

**Comment by SomeoneToIgnore** [comment]
You're looking for `outline::Toggle` command that will open the panel from the issue linked.
There's also an outline panel (next command in the screenshot) which soon will be able to filter entries too but for now can show the same symbols in a non-blocking way.

![image](https://github.com/zed-industries/zed/assets/2690773/94df7364-8faa-4087-b5e8-c83dd1bf5e8b)


Not the first time it gets confusing, so I wonder what would be a good combination of names for all of these.

**Reply by Blankeos** [reply]
Thanks!
